### ISSUE TITLE
Execution context was destroyed, most likely because of a navigation.

### ISSUE THREAD

**Issue by JDZW2014** [open-wa/wa-automate-nodejs#2131]
### Are you using the latest version of the library?

- [X] I have checked and am using the latest version of the library.

### Current Behavior

login whatsapp but failed

### Expected Behavior

- Injecting api
- WAPI injected
- Authenticating
- Authenticate to continue
[2021-09-15T07:52:17.974Z] (node:7) UnhandledPromiseRejectionWarning: Error: Execution context was destroyed, most likely because of a navigation.
    at rewriteError (/chatbot/lib/wa_automate_nodejs/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:261:23)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async ExecutionContext._evaluateInternal (/chatbot/lib/wa_automate_nodejs/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:161:64)
    at async ExecutionContext.evaluate (/chatbot/lib/wa_automate_nodejs/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:106:16)
[2021-09-15T07:52:17.974Z] (node:7) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
[2021-09-15T07:52:17.974Z] (node:7) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

### Mode

My own code

### create() code

```markdown
wa.create({authTimeout: 0}).then(...)
```

### DEBUG INFO

```markdown
{
  "WA_VERSION": "2.2136.9",
  "WA_AUTOMATE_VERSION": "4.21.0",
  "BROWSER_VERSION": "HeadlessChrome/87.0.4272.0",
  "START_TS": 1631692321769
}
```

### Environment

```markdown
- OS: Linux 5.4
- Node: 12.21.0
- npm:
```

**Comment by blackchineykh** [comment]
Try deleting your existing session file/data if you have one

**Comment by rn0x** [comment]
```bash
Error: Execution context was destroyed, most likely because of a navigation.
    at rewriteError (C:\Users\null\Desktop\wa-automate\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:265:23)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async ExecutionContext._evaluateInternal (C:\Users\null\Desktop\wa-automate\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:219:60)
    at async ExecutionContext.evaluate (C:\Users\null\Desktop\wa-automate\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:110:16)
Browser page has closed. Killing client
Killing client. Shutting Down
Error while terminating browser PID. You can just ignore this, as the process has most likely been terminated successfully already: kill ESRCH
Error while terminating browser PID. You can just ignore this, as the process has most likely been terminated successfully already: kill ESRCH
Error: Execution context was destroyed, most likely because of a navigation.
    at rewriteError (C:\Users\null\Desktop\wa-automate\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:265:23)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async ExecutionContext._evaluateInternal (C:\Users\null\Desktop\wa-automate\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:219:60)
    at async ExecutionContext.evaluate (C:\Users\null\Desktop\wa-automate\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:110:16)

```


create

```js

const options = {
    multiDevice: true,
    authTimeout: 0,
    blockCrashLogs: true,
    useChrome: true,
    autoRefresh: true,
    cacheEnabled: true,
    qrRefreshS: 0,
    throwErrorOnTosBlock: false,
    deleteSessionDataOnLogout: false,
    skipUpdateCheck: false,
    bypassCSP: true,
    headless: true,
    logConsole: false,
    //executablePath: "/usr/bin/google-chrome-stable", //  اذا كان نظامك لينكس قم بإلغاء التعليق من هذا السطر // linux
    //executablePath: "C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe", //  اذا كان نظامك ويندوز قم بإلغاء التعليق من هذا السطر // windows
    qrTimeout: 0,
    sessionId: 'Bot_Adhkar'
};

wa.create(options)
.then(async client => await Bot_Adhkar(client))
.catch(e => console.log(e)) 

```


```bash

┌─────────────────────┬──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│       (index)       │                                                                    Values                                                                    │
├─────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│     WA_VERSION      │                                                                 '2.2208.14'                                                                  │
│       PAGE_UA       │ 'WhatsApp/2.2147.16 Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36' │
│ WA_AUTOMATE_VERSION │                                                     '4.32.14 UPDATE AVAILABLE: 4.32.15'                                                      │
│   BROWSER_VERSION   │                                                        'HeadlessChrome/99.0.4844.74'                                                         │
│         OS          │                                                                 'Windows 10'                                                                 │
│      START_TS       │                                                                1647615606390                                                                 │
└─────────────────────┴────────────────────────────────────────────────────────────────────────────────────────────────────────────────

```
